packages/build-essentials/src/styles/styleConstants.js

```diff
@@ -29,7 +29,7 @@ const config = {
         primaryToolbar: ['context'],
         checkboxInput: ['context'],
         dropdownContents: ['context'],
-        selectBoxContents: ['context'],
+        selectBoxContents: '3',
```

Comment:
Yep, this is OK for now, the z-index concept is kind of broken now, needs re-thinking

Reply:
we have an own issue for that afaik :P